assets/scss/base/_lists.scss

```diff
@@ -111,6 +190,21 @@ ul, nav ul {
 	}
 }
 
+/*
+Action list
+
+An action list `.action-list` with each element having a top border
+
+Markup:
+<ul class="action-list">
+  <li><p>First paragraph<p><p>Second paragraph</p></li>
```

Comment:
Can you reformat this for better legibility? Each tag on a new line and indented